move-mutation-test/README.md

```diff
@@ -157,17 +174,20 @@ _In below examples, the `RUST_LOG` flag is used to provide a more informative ou
 To use the tool on only the `Operators` module for the project `simple`, run:
 ```bash
 RUST_LOG=info ./target/release/move-mutation-test run --package-dir move-mutator/tests/move-assets/simple -o report.txt --move-2 --mutate-modules Operators
+./target/release/move-mutation-test display-report -p report.txt --modules Operators
```

Comment:
nit: we've not added the CLI dump for `display-report` subcommand. It only includes a `run` version.

Reply:
As we talked about F2F (finally 😄), let's remove all help outputs to make it more clear.
Since anyone will, by default, use that option to explore more.